spec/unit/knife/core/bootstrap_context_spec.rb

```diff
@@ -20,6 +20,10 @@
 require "chef/knife/core/bootstrap_context"
 
 describe Chef::Knife::Core::BootstrapContext do
+  before do
+    Chef::Config[:fips] = false
```

Comment:
If this is the default, you shouldn't have to do this. We reset `Chef::Config` before every test via a thing in `spec_helper`